.github/workflows/build-artifacts.yml

```diff
@@ -54,61 +74,75 @@ jobs:
       dry-run: false
       oidc-provider-name: database-gh-aerospike
       oidc-audience: database-gh-aerospike
-      jf-build-name: ${{ github.event.repository.name }}
+      jf-build-name: "aerospike-asadm-${{ needs.get-version.outputs.VERSION }}"
```

Comment:
Version should not be in the build name. The build name shouldn't change for each build (?so you can look at a build name and see the list of builds which would be the ids)